Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
-black = "==19.10b0"
+black = "==22.8.0"
```

Comment:
I suggest no longer pinning, we shoud use latest.

Reply:
No longer pinned 👍